test/jdk/javax/swing/JComboBox/6559152/bug6559152.java

```diff
@@ -99,9 +99,6 @@ private static void setupUI() {
     private static void test() throws Exception {
         robot.mouseMove(p.x, p.y);
```

Comment:
Are we still need this? There is no more mouse click.

Reply:
I re-added the mouse click and took Prasanta's advice to just center the mouse on the combobox, instead of adding to the y position by 1.